client/src/components/TaskOptionsModal.jsx

```diff
@@ -1,13 +1,24 @@
 // import {React, useState} from 'react';
 import React, { useState, useEffect, useRef } from 'react';
-import Button from '@mui/material/Button';
-import TextField from '@mui/material/TextField';
-import Modal from '@mui/material/Modal';
+import moment from 'moment';
+import { Button, TextField, Modal, Stack, InputLabel, Select, MenuItem, Container } from '@mui/material';
 
-var minutes = [15, 30, 45, 60];
-var hours = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
-var days = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30];
```

Comment:
This is ugly stuff I'm glad we don't need. Where are these vals coming from now?

Reply:
Was implementing something different, but eventually went with DatePicker that Meredith was using.